docs/source/cpp/compute.rst

```diff
@@ -1728,6 +1728,18 @@ overflow is detected.
 
 * \(2) :member:`CumulativeOptions::start` is ignored.
 
+Other numeric functions
```

Comment:
Maybe we can categorize it as "statistics functions", as the cc file name suggests?

Reply:
I'm going with "statistical functions" then :)